carball/analysis/events/bump_detection/bump_analysis.py

```diff
@@ -0,0 +1,39 @@
+import pandas as pd
+from carball.generated.api.stats.events_pb2 import Bump
+
+from carball.generated.api.player_id_pb2 import PlayerId
+from carball.json_parser.game import Game
+
+from carball.generated.api import game_pb2
+
+
+class BumpAnalysis:
+    def __init__(self, game: Game, proto_game: game_pb2):
+        self.proto_game = proto_game
+
+    def get_bumps_from_game(self, data_frame: pd.DataFrame):
+        self.create_bumps_from_demos(self.proto_game)
+
+        self.analyze_bumps(data_frame)
+
+    def create_bumps_from_demos(self, proto_game):
+        for index in range(len(proto_game.game_metadata.demos)):
```

Comment:
very minor, but I believe you can just do a `for demo in proto_game.game_metadata.demos`. same in the analyze_bumps function below